examples/zupass-feed-server/src/feeds.ts

```diff
@@ -0,0 +1,168 @@
+import { EdDSAPCDPackage } from "@pcd/eddsa-pcd";
+import {
+  EdDSATicketPCD,
+  EdDSATicketPCDPackage,
+  ITicketData
+} from "@pcd/eddsa-ticket-pcd";
+import { EmailPCDPackage } from "@pcd/email-pcd";
+import {
+  FeedHost,
+  PollFeedRequest,
+  PollFeedResponseValue,
+  verifyFeedCredential
+} from "@pcd/passport-interface";
+import {
+  DeleteFolderPermission,
+  PCDAction,
+  PCDActionType,
+  PCDPermissionType,
+  ReplaceInFolderPermission
+} from "@pcd/pcd-collection";
+import { ArgumentTypeName, SerializedPCD } from "@pcd/pcd-types";
+import { SemaphoreSignaturePCDPackage } from "@pcd/semaphore-signature-pcd";
+import _ from "lodash";
+import path from "path";
+import { Ticket, loadTickets } from "./config";
+import { ZUPASS_PUBLIC_KEY } from "./main";
+
+const fullPath = path.join(__dirname, "../artifacts/");
+SemaphoreSignaturePCDPackage.init?.({
+  zkeyFilePath: fullPath + "16.zkey",
+  wasmFilePath: fullPath + "16.wasm"
+});
+
+EdDSAPCDPackage.init?.({});
+EdDSATicketPCDPackage.init?.({});
```

Comment:
nit: this calling convention here is a little concerning as these are asynchronous functions, and can take indeterminate amounts of time or reject, leaving the application in an undefined state.